Documentation/gettingstarted/hubble-configuration.rst

```diff
@@ -24,212 +24,272 @@ This allows Hubble Relay to communicate with all Hubble instances in the
 cluster. Connections between Hubble instances and Hubble Relay are secured using
 mutual TLS (mTLS) by default.
 
-TLS certificates can be provided by manually on the Helm install command (user
-provided) or generate automatically via either:
+TLS certificates can be generate automatically or provided by manually.
+
+The following options are available to configure TLS certificates automatically:
 
-* `Helm <https://helm.sh/docs/chart_template_guide/function_list/#gensignedcert>`__
-* cilium's `certgen <https://github.com/cilium/certgen>`__ (using a Kubernetes ``CronJob``)
 * `cert-manager <https://cert-manager.io/>`__
+* cilium's `certgen <https://github.com/cilium/certgen>`__ (using a Kubernetes ``CronJob``)
+* `Helm <https://helm.sh/docs/chart_template_guide/function_list/#gensignedcert>`__
```

Comment:
nit: do we maybe want to put the certgen method first as it's arguably the easiest of the two to implement?

Reply:
It's easiest yes, but arguably cert-manager is the best of the options and I think we should encourage it's usage.